test/CMakeLists.txt

```diff
@@ -19,7 +24,7 @@ macro(add_ledger_harness_tests _class)
     file(GLOB ${_class}_TESTS *.test)
     foreach(TestFile ${${_class}_TESTS})
       get_filename_component(TestFile_Name ${TestFile} NAME_WE)
-      string(FIND ${TestFile_Name} "_py.test" TestFile_IsPythonTest)
+      string(FIND ${TestFile} "_py.test" TestFile_IsPythonTest)
```

Comment:
The `TestFile_Name` variable contains the _"File name with neither the directory nor the longest extension" (see [documentation for `get_filename_component`](https://cmake.org/cmake/help/latest/command/get_filename_component.html)), so either the string to search for should be `"_py"` or the variable to search should be `${TestFile}`.